### ISSUE TITLE
Backups location has changed

### ISSUE THREAD

**Issue by atrepca** [tmonck/clean_up_snapshots#28]
Hi,

Thanks for writing this add-on. The backups location has changed in the latest Home Assistant, I'm currently on:
```
Home Assistant 2022.12.1
Supervisor 2022.11.2
Operating System 9.3
Frontend 20221208.0 - latest
```
The backups dir is now `/backup` not `/backups`, and so the add-on does nothing when running.
```sh

| |  | |                          /\           (_)   | |            | |  
| |__| | ___  _ __ ___   ___     /  \   ___ ___ _ ___| |_ __ _ _ __ | |_ 
|  __  |/ _ \| '_ \ _ \ / _ \   / /\ \ / __/ __| / __| __/ _\ | '_ \| __|
| |  | | (_) | | | | | |  __/  / ____ \\__ \__ \ \__ \ || (_| | | | | |_ 
|_|  |_|\___/|_| |_| |_|\___| /_/    \_\___/___/_|___/\__\__,_|_| |_|\__|

Welcome to the Home Assistant command line.

System information
  IPv4 addresses for eth0:  <REMOVED>
  IPv6 addresses for eth0:  <REMOVED>
  IPv4 addresses for wlan0: 

  OS Version:               Home Assistant OS 9.3
  Home Assistant Core:      2022.12.1

  Home Assistant URL:       http://homeassistant.local:8123
  Observer URL:             http://homeassistant.local:4357

➜  ~ 
➜  ~ find / -name 'backup*' -type d
/backup
```

This is not configurable, but maybe it should be just in case this changes again.

Thanks,

**Comment by tmonck** [comment]
So I don't look at the file system at all I am making API calls to home-assistant. I haven't upgrade yet but was going tomorrow. I will upgrade and either fix or let you know if I am unable to replicate.

**Comment by atrepca** [comment]
False alert, I didn't have the REST API enabled, that was probably the cause.

I was able to confirm it works with a `curl` call:
```bash
curl -skX GET -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" https://homeassistant.local:8123/api/hassio/backups
```

I also didn't have HACS logs enabled and didn't know to check `/config/home-assistant.log` for troubleshooting, which made it hard (still learning this HA thing).

Confirmed this works as expected from an automation, all good, thanks and sorry for the false alert.
